protocol/daemons/liquidation/client/sub_task_runner.go

```diff
@@ -50,6 +52,16 @@ func (s *SubTaskRunnerImpl) RunLiquidationDaemonTaskLoop(
 		return err
 	}
 
+	if lastCommittedBlockHeight == s.lastLoopBlockHeight {
```

Comment:
IIUC if application restarts, `lastLoopBlockHeight` is by default `0` and we always continue below logic. If so could we document?